ansible/create-aws-instance.yml

```diff
@@ -0,0 +1,15 @@
+---
+#
+# Example Usage:
+#
+# time ansible-playbook ansible/aws.yml -e "aws_create_all=true"
```

Comment:
I can't seem to find where/how this var `aws_create_all` interacts so far, perhaps we need the latest commits to your work here.